lib/adapter.js

```diff
@@ -26,7 +26,7 @@ var metrics = {
 
 var internal = {
   on: internal_on, // XXX(sam) where is this used?
-  supports: {watchdog: false} // XXX(sam) probably not needed
+  supports: {watchdog: process.platform === 'linux'}
```

Comment:
its better if this information can come from appmetrics directly, so if you add support for some other platform, or if support doesn't work on linux/ppc (random example), then no changes are required in strong-supervisor